streams/src/main/java/org/apache/kafka/streams/kstream/internals/MaterializedInternal.java

```diff
@@ -26,6 +26,8 @@
 
 public class MaterializedInternal<K, V, S extends StateStore> extends Materialized<K, V, S> {
 
+    private static final String STATE_STORE_SUFFIX = "-store";
```

Comment:
Not in the KIP, we'll drop this, plus users already have full control over the state store name, so this could introduce a non-backward compatible change.